index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Microverse Technical Document</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <nav id="navbar">
+    <header><h1>JS Documentation</h1></header>
+    <a href="#Introduction" class="nav-link">Introduction</a>
+    <a href="#What_you_should_already_know" class="nav-link">What you should already know</a>
+    <a href="#JavaScript_and_Java" class="nav-link">Javascript and Java</a>
+    <a href="#Hello_world" class="nav-link">Hello World</a>
+    <a href="#Variables" class="nav-link">Variables</a>
+    <a href="#Declaring_variables" class="nav-link">Declaring Variables</a>
+    <a href="#Variable_scope" class="nav-link">Variable Scope</a>
+    <a href="#Global_variables" class="nav-link">Global Variables</a>
+    <a href="#Constants" class="nav-link">Constants</a>
+  </nav>

```

Comment:
For semantic HTML, &#60;header&#62; is the first part of the body, so I think your html will more semantic if you put &#60;nav&#62; inside &#60;header&#62; part.

Reply:
### [OPTIONAL] class "nav-link" may be  removed here, if we use chain selection #navbar a {...} instead of using .nav-link {...}

Reply:
[OPTIONAL] This is optional but recommended I think the previous reviewer :point_up_2: missed this comment so kindly explain more about semantic tags and `why` we use semantic tags then the student understand easily :slightly_smiling_face: